apps/fabric-website/index.html

```diff
@@ -57,7 +60,9 @@
 
     // If the site is hosted, load the flight config script for production or dogfood
     if (!isLocal) {
-      var configPrefix = isProduction ? 'fabric-website-prod' : 'fabric-website-df';
+      var v5 = fabricVersion == '5' ? 'v5-' : '';
```

Comment:
```suggestion
      var v5 = fabricVersion == '5' ? 'v5-' : '';
```

Prefer `===` unless there's a reason not to.